assets/src/story-embed-block/block/save.js

```diff
@@ -53,6 +53,7 @@ function save({ attributes }) {
         poster={poster}
         width={width}
         height={height}
+        align={align}
```

Comment:
Can you elaborate on this change? How does it relate to the rest of the PR?

Note that changes to a block's `save` method usually warrant proper [block deprecations](https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/).

Reply:
^ This is why you were getting block validation errors

Reply:
I got the errors before I added the prop not after. After it was fixed.

Reply:
Since the JS changes are unrelated to this issue, can you move that to a new PR and open an issue about it?

Again, any changes to the `save` method requires block deprecations, so this needs some thorough testing.

Reply:
It is related, as the markup of the block has changed to include a `margin:0px`. Nothing has been deprecated something has been added. I am confused what your point is here.